### DISCUSSION TITLE
CoPilot entering in extra endings for the code that I am working on

### DISCUSSION THREAD

**Discussion by ollie-iterators** [community/community#14957] [Copilot Conversations]
When I am doing things like working on writing a multiline comment with Python, it sometimes tries to add on an extra """ to the end of the comment, even though I am already doing that.

Also, when I am working with JavaScript, it sometimes wants to add an extra "}" to a for loop or an if statement that I am working on writing.

**Comment by Kek5chen** [comment]
I have also seen that happen a couple of times now in my C++ code.
There currently is nothing we can do to fix this, but, as you might know, github copilot is actively in development and i suggest that patience is key for this issue.
